R/polish_emissions_profile_company.R

```diff
@@ -9,7 +9,7 @@ polish_emissions_profile_company <- function(ep_comp, ep_prod, europages_compani
   ep_prod <- sanitize_isic(ep_prod)
 
   inter_result <- prepare_inter_emissions_profile(ep_prod, europages_companies, ecoinvent_activities, ecoinvent_europages, isic) |>
-    select("companies_id", "company_name", "company_city", "country", "postcode", "address", "main_activity", "avg_matching_certainty") |>
+    select("companies_id", "company_name", "company_city", "country", "postcode", "address", "main_activity", "avg_matching_certainty", "min_headcount", "max_headcount") |>
```

Comment:
If you're note sure the data will have these columns, then you may use `matches(c("min_headcount", "max_headcount"))`.

Reply:
They should be in it. Thanks for the suggestion!